protokube/pkg/protokube/aws_volume.go

```diff
@@ -164,73 +164,83 @@ func newEc2Filter(name string, value string) *ec2.Filter {
 func (a *AWSVolumes) findVolumes(request *ec2.DescribeVolumesInput) ([]*Volume, error) {
```

Comment:
That is fine, I will extract a third method for unit testing then. I did not mock out the unit test with the aws mocks, so that is why I left the aws code here, and extracted a func that I could test.